rows.go

```diff
@@ -22,7 +22,14 @@ const (
 )
 
 var (
-	maxChunkDownloadWorkers        = 10
+	// MaxChunkDownloadWorkers specifies the maximum number of goroutines used to download chunks
+	MaxChunkDownloadWorkers = 10
```

Comment:
Now the application can change the number of chunk download workers. This won't help much for the standard JSON decoder to reduce memory footprint, though.